packages/docusaurus-theme-translations/locales/base/theme-common.json

```diff
@@ -45,6 +45,7 @@
   "theme.blog.archive.description___DESCRIPTION": "The page & hero description of the blog archive page",
   "theme.blog.archive.title": "Archive",
   "theme.blog.archive.title___DESCRIPTION": "The page & hero title of the blog archive page",
+  "theme.blog.author.noPosts": "This author has not written any posts yet.",
```

Comment:
I suspect you didn't run the command the CI / tests tells you to run, this is not the expected result from running it.

What did you run?

Reply:
`yarn jest`, what am I supposed to run ?

Reply:
The tests are now passing, but you didn't run the message that the previously failing tests printed:

https://github.com/facebook/docusaurus/actions/runs/10473792292/job/29006473892?pr=10431

> Errors thrown in packages/docusaurus-theme-translations/locales/__tests__/locales.test.ts
> Summary of all failing tests
> FAIL packages/docusaurus-theme-translations/locales/__tests__/locales.test.ts
>   ● theme translations › has base messages files contain EXACTLY all the translations extracted from the theme. Please run "yarn workspace @docusaurus/theme-translations update" to keep base messages files up-to-date

I can see this because your extracted message has no description, and it only modified the base locale, not all the other localized files.

You are not supposed to enter that message manually in theme translations: it's the CLI command that it supposed to do so